release-team/shadows.md

```diff
@@ -0,0 +1,92 @@
+
+## Week 1
+
+The first week should be used to get Shadows set up with all of the "paperwork" they need in order to participate fully in the Release Team:
+
+* Sign the [CNCF CLA](https://github.com/kubernetes/community/blob/master/CLA.md), which they need for all contributions.
+* Apply to be an Org Member if they are not already (see below).
+* Join the [#sig-release slack channel](https://slack.k8s.io/).
+* Get added to the `kubernetes-release-team` group by the Release Lead.
+* Join the [sig-release](https://groups.google.com/forum/#!forum/kubernetes-sig-release) and [release-team](https://groups.google.com/forum/#!forum/kubernetes-release-team) Google Groups and calendars.
+* All all contact information to their Release Team's contact sheet.
+* Get invited to the weekly Release Team Meeting and add the [sig-release calendar](https://calendar.google.com/calendar/embed?src=kipmnllvl17vl9m98jen6ujcrs%40group.calendar.google.com) to their calendar
+
+Becoming a Release Team Shadow is considered sufficient contribution to become a Kubernetes Org Member, and membership is needed for several tasks, especially having the Github bots obey the Shadow's commands. As such, each Shadow who is not already an org member should [apply to become one](https://github.com/kubernetes/community/blob/master/community-membership.md#member) with their mentoring role lead, the Release Lead, and/or the Emeritus Advisor as their sponsors.
+
+Most times, groups of shadows and their lead for a specific role will have a meeting during the first two weeks.   Not all do, though, since sometimes there's a considerable spread of time zones, and some leads prefer to work asynchronously.
+
+## Weeks 2-8
+
+For the "main" part of the development cycle, new shadows should be learning their roles and starting to take on tasks for their roles. This may include:
+
+* Attending the weekly Release Team meeting, and possibly delivering the report for their Role for the week.
+* Checking on daily statuses of things in their roles and recording them in documents shared by the team.
+* Contacting and sending reminders to SIGs.
+* Reviewing and resolving open issues in sig-release and test-infra that affect their roles, especially ones left over from last release.
+* Other tasks as defined in the [Role Handbook](https://github.com/kubernetes/sig-release/tree/master/release-team).
+
+For a couple roles, particularly Enhancements, this first period is actually the busiest period of the release, so there will be a lot for the Shadows to do.  For other roles, the shadows for a role should expect to be in communication with their Lead and each other a few times a week, via Slack, shared documents, and Github issues/project boards.
```

Comment:
maybe worth adding something about asking the role lead when is the bushiest time of the release cycle as something shadows should do in the early weeks?

Reply:
better, say "consult the handbook". will add.